docs/standard/exceptions/toc.yml

```
@@ -19,3 +19,5 @@
     href: handling-com-interop-exceptions.md
   - name: Best Practices for Exceptions
     href: best-practices-for-exceptions.md
+  - name: "How to: Create custom exceptions and localized exception messages"
```

This should be moved up to follow **How to: Create user-defined exceptions**.
```suggestion
  - name: "How to: Create user-defined exceptions and localized exception messages"
```